src/main/java/jenkins/plugins/logstash/LogstashBuildWrapper.java

```diff
@@ -24,100 +24,63 @@
 
 package jenkins.plugins.logstash;
 
+import java.io.IOException;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+
 import hudson.Extension;
 import hudson.Launcher;
-import hudson.model.BuildListener;
 import hudson.model.AbstractBuild;
 import hudson.model.AbstractProject;
-import hudson.model.BuildableItemWithBuildWrappers;
+import hudson.model.BuildListener;
 import hudson.tasks.BuildWrapper;
 import hudson.tasks.BuildWrapperDescriptor;
 
-import java.io.IOException;
-import java.io.OutputStream;
-import java.util.ArrayList;
-import java.util.List;
-
-import org.kohsuke.stapler.DataBoundConstructor;
-
-import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsBuildWrapper;
-import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsBuildWrapper.VarPasswordPair;
-import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsConfig;
-
 /**
- * Build wrapper that decorates the build's logger to insert a
+ * Build wrapper that only indicates to the ConsoleLogFilter
+ * that it should forward log to an indexer. Keep this for backward compatibility.
+ * Better approach would be a JobProperty.
```

Comment:
Can you please add a new issue to track this?

Reply:
opened JENKINS-49960